### DISCUSSION TITLE
What is requestedPublishingInterval when creating a subscription ?

### DISCUSSION THREAD

**Discussion by argiraud** [eclipse-milo/milo#848] [General]
Hi,
To create a subscription we need to have an attribute _requestedPublishingInterval_ with the method _createSubscription_ but I don't understand which double value I should put.
Someone can help me to figure what this value is about pls

Thank you

**Comment by kevinherron** [comment]
Here's an excerpt from Part 4 of the OPC UA spec, which you should read to fully understand how subscriptions work:

> Subscriptions have a publishing interval. The publishing interval of a Subscription defines the cyclic rate at which the Subscription executes. Each time it executes, it attempts to send a NotificationMessage to the Client. NotificationMessages contain Notifications that have not yet been reported to Client.

**Reply by argiraud** [reply]
Thank you I will check the OPC UA spec
